src/service/session.py

```diff
@@ -1,6 +1,27 @@
 from os import environ
+import jwt
 
 if "BASE_URL" not in environ:
     raise Exception("Please add BASE_URL to env")
 BASE_URL = environ["BASE_URL"]
 print("Make sure to change BASE_URL in your env when generating a new one or restarting localstack.")
+
+
+TOKEN_FILENAME = "user_token.txt"
+
+
+def get_username() -> str:
+    token = ""
+    try:
+        with open(TOKEN_FILENAME, "r") as token_file:
+            token = token_file.readline().strip()
+    except FileNotFoundError:
+        pass
```

Comment:
Call `get_jwt()` instead of manually reading it from file?

Reply:
[ecc2193](https://github.com/magley/klau-drive/pull/24/commits/ecc21939d394dd3571d65504dcb0136e09d0510a)